CONTRIBUTORS_QUICK_START.rst

```diff
@@ -544,6 +551,48 @@ Setting up Debug
                alt="Add environment variable pycharm">
         </div>
 
+- Now Debug an example dag and view the entries in tables such as ``dag_run, xcom`` etc in mysql workbench.
+
+B. Using Visual Studio Code
+
+- In Visual Studio Code open airflow project, directory ``\files\dags`` of local machine is by default mounted to docker
+  machine when breeze airflow is started. So any DAG file present in this directory will be picked automatically by
```

Comment:
I think this paragraph is a bit confusing. Is there a central thing you're trying to communicate?

Reply:
This paragraph is duplicated from the **A. Using PyCharm** part to keep consistency.